helix-term/src/application.rs

```diff
@@ -42,7 +49,7 @@ pub struct Application {
     callbacks: LspCallbacks,
 
     lsp_progress: LspProgressMap,
-    lsp_progress_enabled: bool,
+    lsp_display_messages: bool,
```

Comment:
One last change, let's store `Config` as `application.config`. That way we're not pulling variables out manually and we can later simply replace the struct to do a config reload.

Reply:
Again, much cleaner. Done